pkg/types/wallet.go

```diff
@@ -41,10 +41,10 @@ const (
 
 // WalletInfo single wallet record
 type WalletInfo struct {
-	Data string      `json:"data"`
 	ID   uint32      `json:"id"`
 	Name string      `json:"name"`
 	Type *WalletType `json:"type"`
+	Data string      `json:"data"`
```

Comment:
Was this intentionally left `string`?

Reply:
Yep. Apparently in this case its json encoded string https://github.com/Chia-Network/chia-blockchain/blob/main/chia/wallet/wallet_info.py#L26